independent-publisher-connectors/Hash Generator/apiProperties.json

```diff
@@ -0,0 +1,13 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",

```

Comment:
Please modify the iconbrandcolor as per the template - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json